ignite/contrib/metrics/fid.py

```diff
@@ -0,0 +1,95 @@
+import os
+from typing import Callable, Sequence, Union
+
+import torch
+from torchvision import transforms
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+try:
+    from scipy import linalg
+except ImportError:
+    raise RuntimeError("This contrib module requires scipy to be installed.")
+
+
+__all__ = ["FID"]
+
+
+class FID(Metric):
+    """Calculates FID metric
+
+    """
+
+    def __init__(
+        self, output_transform: Callable = lambda x: x, fid_model=None,
```

Comment:
```suggestion
        self, output_transform: Callable = lambda x: x, fid_model: nn.Module = None,
```

Reply:
Recently, we reintroduced `device` argument to store internal vars as tensors: https://github.com/pytorch/ignite/blob/master/ignite/metrics/metric.py#L131, we have to add it here too. Please, take a look at how other metrics are implemetned, e.g. https://github.com/pytorch/ignite/blob/master/ignite/metrics/accuracy.py#L96